### DISCUSSION TITLE
Seeking Insights on Verifying NEON Utilization in NCNN Models for R-Car S4 ARM-based Device.

### DISCUSSION THREAD

**Discussion by pradnya720** [Tencent/ncnn#5505] [General]
Hi everyone,

I am currently working on optimizing NCNN models for deployment on the R-Car S4, an ARM-based aarch64 device running QNX OS. As part of this optimization process, I am keen to verify whether the NCNN models are effectively utilizing ARM NEON instructions on this platform.

I would greatly appreciate any insights, experiences, or tools that you may have for assessing NEON utilization in NCNN models. If you have encountered similar challenges or have expertise in optimizing NCNN for ARM architectures like the R-Car S4, your input would be immensely valuable to me.

Thank you in advance for your assistance!

Looking forward to your responses.

Best regards,
Pradnya Unhale

**Comment by nihui** [comment]
If your toolchain target is aarch64, arm neon will always be enabled in ncnn.
You may add some printf in between ncnn arm neon codes to see if it would output :)
